.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -25,12 +25,17 @@ If applicable, add screenshots to help explain your problem.
 
 **Please complete the following information:**
 
-- OS: [e.g. iOS]
+- Operating System: [e.g. Windows 11]
```

Comment:
Os Naming Can be such as 
Windows (Version) , Mac (Version) , Linux (Name)(Version)